src/redshift-gtk/statusicon.py

```diff
@@ -29,9 +29,13 @@
 import re
 import gettext
 
+import gi
+gi.require_version('Gtk', '3.0')
+
 from gi.repository import Gtk, GLib, GObject
 
 try:
+    gi.require_version('AppIndicator3', '0.1')
```

Comment:
I'm not sure if this is going to work. It seems to me that a `ValueError` is raised when AppIndicator3 is not available.